### PR TITLE
Autogenerate documented REPL banner

### LINKED ISSUE #28584: REPL Documentation outdated for 0.7/1.0
Documentation for 0.7/1.0 both include a console 'screen-shot' of 0.6 and do not have any mention of the new Pkg3 REPL mode that can be entered through ']'.

### PR DESCRIPTION
**Pull request by HarrisonGrodin**
fixes  #28584

### CODE REVIEW

File: stdlib/REPL/docs/src/index.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ $ julia
...
+   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
   | | | | | | |/ _` |  |
-  | | |_| | | | (_| |  |  Version 0.6.0-dev.2493 (2017-01-31 18:53 UTC)
- _/ |\__'_|_|_|\__'_|  |  Commit c99e12c* (0 days old master)
+  | | |_| | | | (_| |  |  Version 1.0.0-rc1.5 (2018-08-07 20:49 UTC)
+ _/ |\__'_|_|_|\__'_|  |  Commit d038f2f* (0 days old master)
 |__/                   |  x86_64-linux-gnu
 
-julia>
+julia>
```
Comment (KristofferC) [inline stdlib/REPL/docs/src/index.md]: Trailing whitespace I think

Comment (HarrisonGrodin) [reply stdlib/REPL/docs/src/index.md]: Ouch, good catch. Fixed.

**Comment by fredrikekre** [comment]
You can instead do what we do at the top of  https://github.com/JuliaLang/julia/blob/master/doc/src/manual/getting-started.md and generate it at build time.

**Comment by waldyrious** [comment]
For future reference, this PR updates the docs to match the changes made in #28522 and #27231.

**Pull request merged**